docs/metrics/index.md

```diff
@@ -13,6 +10,15 @@ Each ML use case requires different metrics. Using the right metrics is critical
 comparing model performance. In each metrics guide, you can learn about the metric with examples, its limitations and
 biases, and its intended uses.
 
+<div class="grid cards" markdown>
+- [Accuracy](accuracy.md)
+
+    ---
+
+    Accuracy measures how well a model predicts correctly. It's a good metric for assessing model performance in simple
+    cases with balanced data.
+</div>
+
 <div class="grid cards" markdown>
```

Comment:
Since there are lots of cards here, it'd be nice to use the grid rather than having each entry take up a whole row. Deleting all of these `<div>`s but the outmost one yields a view like this:

<img width="1904" alt="Screen Shot 2023-06-23 at 4 24 13 PM" src="https://github.com/kolenaIO/kolena/assets/9648886/8bb46b4e-1de7-4816-b0ad-c29e88e8199c">